### DISCUSSION TITLE
Way to ignore some files from the database. Similar to Obsidian Projects plugin.

### DISCUSSION THREAD

**Discussion by AzadKshitij** [RafaelGB/obsidian-db-folder#866] [Q&A] [answered]
### Contact Details

azadkshitij08302001@gmail.com

### Present your request

I want a way to ignore files I don't want to include in my database.

![image](https://user-images.githubusercontent.com/54288405/224187942-9ed98edf-abeb-434d-87a3-fe6bedabd2ec.png)

### For which platform do you request this request??

Desktop

**Comment by RafaelGB** [comment, marked as answer]
Hello @AzadKshitij , you can use the dbfolder filters. In your case a filter with "tags is not empty" could be enough 

<img width="580" alt="Screenshot 2023-03-20 at 22 52 01" src="https://user-images.githubusercontent.com/11924043/226473681-ba6eed82-ad61-4233-8bf5-722181ee041d.png">

**Comment by acook** [comment]
This is a really nice feature!

I do wish there was a way to automatically ignore Folder Notes like those used by Ozan's File Tree plugin or Make.md, etc. 

In the meantime I use some kind of placeholder value in a column so the filter can tells the DB to ignore it.
